lib/tasks/readyset.rake

```diff
@@ -69,4 +141,22 @@ namespace :readyset do
       end
     end
   end
+
+  desc 'Prints status information about ReadySet'
+  task status: :environment do
+    rows = Readyset.raw_query('SHOW READYSET STATUS').
+      map { |result| [result['name'], result['value']] }
+    table = Terminal::Table.new(rows: rows)
+
+    puts table
+  end
+
+  desc 'Prints information about the tables known to ReadySet'
+  task tables: :environment do
+    rows = Readyset.raw_query('SHOW READYSET TABLES').
+      map { |result| [result['table'], result['status'], result['description']] }
+    table = Terminal::Table.new(headings: [:table, :status, :description], rows: rows)
+
+    puts table
+  end
 end
```

Comment:
:sunglasses: : I can definitely see these rake tasks getting a lot of use. Nice, nice, nice.